ogarniacz_mp3.py

```diff
@@ -113,7 +113,19 @@ def downloaded_to_processing(source_dir_path, target_dir_path): #ścieżka do ka
                 except BadZipfile:
                     print(f"Plik {f} nie jest plikiem .zip.")
 
-                    
+def getting_objects(processing_dir_path):
+    dir_objects = []
+    file_objects = []
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path, topdown=False):
+        music_dir = classes.MusicDir(dir_path)
+        dir_objects.append(music_dir)
+        for file in files:
+            music_file = classes.MusicFile(file)
```

Comment:
`classes.MusicFile` powinien przyjmować ścieżkę do pliku, a nie tylko jego nazwę.
Dodatkowo nie powinnaś tworzyć obiektów MusicFile dla plików które nie są plikami muzycznymi.

Ale to bez znaczenia, bo twoja klasa MusicDir i tak już [tworzy sobie listę własnych plików muzycznych](https://github.com/Bialka/cokolwiek/blob/master/classes.py#L47), więc lista file_objects nie jest ci potrzebna..